src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.Aot.targets

```diff
@@ -137,50 +137,14 @@ They run in a context of an inner build with a single $(RuntimeIdentifier).
         LLVMPath="$(_LLVMPath)"
         LdName="$(_LdName)"
         LdFlags="$(_LdFlags)"
-        CollectTrimmingEligibleMethods="$(AndroidStripILAfterAOT)"
+        CollectTrimmingEligibleMethods="$(_AndroidCollectTrimmingEligibleMethods)"
```

Comment:
I created a private property, `$(_AndroidCollectTrimmingEligibleMethods)`, if someone wants to experiment with this in the future. This way it's possible to set it.